src/IFrameNavigator.ts

```diff
@@ -177,13 +234,13 @@ export default class IFrameNavigator implements Navigator {
 
         let statusMessage = "";
         if (status === CacheStatus.Uncached) {
-            statusMessage = "Not available offline";
+            statusMessage = "Download";
```

Comment:
This seems weird to me, next to the read offline button. Maybe there shouldn't be an additional message in this case.